src/service-card.js

```diff
@@ -0,0 +1,71 @@
+import { LitElement, html, css } from 'lit-element/lit-element.js';
+
+class ServiceCard extends LitElement {
+  static get tag() {
```

Comment:
We ship all tags with a `static get tag` because it's easier to then ask the class "what tag do you create". This is something that the actual specification doesn't require / support but we like using (see bottom of element for it being used then on the class itself)